app/src/main/java/com/appdev/all_in_android/ui/screens/MarketPlaceScreen.kt

```diff
@@ -1,12 +1,179 @@
 package com.appdev.all_in_android.ui.screens
 
-import androidx.compose.foundation.layout.Box
-import androidx.compose.material.Text
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.LazyRow
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Search
+import androidx.compose.material3.Button
+import androidx.compose.material3.ButtonColors
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.Icon
+import androidx.compose.material3.OutlinedTextField
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Text
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.appdev.all_in_android.models.Contract
+import com.appdev.all_in_android.models.ContractRepo
+import com.appdev.all_in_android.ui.components.general.AllInTopBar
+import com.appdev.all_in_android.ui.components.general.PlayerCard
 
+@OptIn(ExperimentalMaterial3Api::class)
 @Composable
-fun MarketplaceScreen(){
```

Comment:
Very minor, but would it be possible to either change the composable name or the file name such that they match each other exactly?

Reply:
done!